spec/features/granules/granule_filters_spec.rb

```diff
@@ -245,65 +246,67 @@
       end
     end
 
-    context "when excluding by granule id" do
-      before :all do
-        first_project_collection.click
-        wait_for_xhr
-        number_granules = granule_list.text.match /Showing \d+ of \d+ matching granules/
-        before_granule_count = number_granules.to_s.split(" ")[3].to_i
-
-        first_granule_list_item.click
-        first_granule_list_item.click_link "Exclude this granule"
-      end
-
-      after :all do
-        click_button "granule-filters-clear"
-        wait_for_xhr
-        granule_list.click_link "Back to Collections"
-        wait_for_xhr
-        first_project_collection.click_link "Show granule filters"
-        wait_for_xhr
-      end
-
-      it "displays an indication that granules have been excluded" do
-        expect(page).to have_content("1 granule has been removed from your results")
-      end
-
-      it "removes the granule from the granule list" do
-        expect(page).to have_css('#granule-list .panel-list-item', count: 38)
-      end
-
-      it "updates the page's hits count" do
-        # 38 = (page size - 1) * 2. Because of the browser height
-        # removing a granule immediately loads the next page.
-        expect(granule_list).to have_content("Showing 38 of #{before_granule_count.to_i - 1} matching granules")
-      end
-
-      context "when the user clicks the link to clear removed granules" do
-        before :all do
-          click_link "Add it back"
-          wait_for_xhr
-        end
-
-        after :all do
-          first_granule_list_item.click
-          first_granule_list_item.click_link "Exclude this granule"
-          wait_for_xhr
-        end
-
-        it "includes the excluded granules in the list" do
-          expect(page).to have_css('#granule-list .panel-list-item', count: 20)
-        end
-
-        it "updates the granule hits count" do
-          expect(page).to have_content("Showing 20 of #{before_granule_count} matching granules")
-        end
-      end
-    end
+    # JS: New layout may make these tests invalid
```

Comment:
Yeah, that is one of my questions in the email. I don't know why these tests are excluded.